detection-rules/attachment_callback_scam.yml

```
@@ -0,0 +1,31 @@
+name: "Callback scam with attachment (first-time sender)"
+description: |
+    An invoice or receipt attachment from a freemail sender which is classified by the NLU engine as a Callback Scam.
```

I think we should drop the phrase "NLU engine" from these in the same way that we don't mention strelka. I personally don't like when rules mention other features, and a user should only have to think about things at a MQL/function level when writing rules.

```suggestion
    Detects callback scams by analyzing text within images of receipts or invoices from first time senders.
```